base/inc/routes/siteorigin-widgets-resource.class.php

```diff
@@ -88,6 +88,11 @@ public function get_widget_form( $request ) {
 
 		/* @var $widget SiteOrigin_Widget */
 		$widget = SiteOrigin_Widgets_Widget_Manager::get_widget_instance( $widget_class );
+		// Attempt to activate the widget if it's not already active.
+		if ( ! empty( $widget_class ) && empty( $widget ) ) {
+			global $wp_widget_factory;
```

Comment:
@AlexGStapleton why are you declaring this global?

Reply:
Shouldn't be. Corrected.